interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -281,6 +281,11 @@ Specification
 *Postconditions*
 
 * The Vault MUST be removed from the ``Vaults`` mapping.
+* The Vault MUST remain above the secure collateralization threshold.
+* ``get_total_nominated_collateral(vault_id)`` must return zero.
+* For all nominators, ``get_nominator_collateral(vault_id, user_id)`` must return zero.
```

Comment:
Are you able to `:ref:` these?

Reply:
Being getters, these were not added to the spec

Reply:
I think `computeRewardAtIndex` should be ref-able right?